src/onedrivesdk/extensions/fragment_upload.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+'''
+The MIT License (MIT)
+
+Copyright (c) 2015 Wiktor Niesiobędzki
```

Comment:
Actually, in the CLA it says you can't keep the copyright. Please copy/paste the license out of one of the other files as-is.